src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -457,47 +465,95 @@ private PaymentData buildPaymentData(final Account account, final Iterable<Plugi
     }
 
     // For API
-    // TODO Generify
```

Comment:
Just for the record, there is one use-case that isn't supported yet (i.e. setting `recurringId`).